tests/unit/AnnualReport.spec.ts

```diff
@@ -1915,19 +1914,13 @@ describe('AnnualReport - Part 6 - Error/Warning Dialogs', () => {
     // mock "file post" endpoint
     const p1 = Promise.reject({
       response: {
-        status: BAD_REQUEST,
+        status: 400,
```

Comment:
It's better to check hard-coded values in the unit tests, so that if `BAD_REQUEST` is ever anything else, the test will let us know (ie, break).